objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestYoloQueries.java

```diff
@@ -432,15 +432,15 @@ public void testImageDateUploadOne() throws InterruptedException, IOException {
                         
         // Checking that all images were uploaded to VANTIQ
         Thread.sleep(1000);
-        checkUploadToVantiq(IMAGE_2.get("filename"), vantiq);
+        checkUploadToVantiq(IMAGE_2.get("filename"), vantiq, VANTIQ_DOCUMENTS);
         
         // Checking that none of the other images were uploaded to VANTIQ
-        checkNotUploadToVantiq(IMAGE_1.get("filename"), vantiq);
-        checkNotUploadToVantiq(IMAGE_3.get("filename"), vantiq);
-        checkNotUploadToVantiq(IMAGE_4.get("filename"), vantiq);
-        checkNotUploadToVantiq(IMAGE_5.get("filename"), vantiq);
-        checkNotUploadToVantiq(IMAGE_6.get("filename"), vantiq);
-        checkNotUploadToVantiq(IMAGE_7.get("filename"), vantiq);
+        checkNotUploadToVantiq(IMAGE_1.get("filename"), vantiq, VANTIQ_DOCUMENTS);
```

Comment:
Life would have been easier if you'd added a 4 parameter overload to `checkNotUpload...()` & `checkUpload...()` :-)

Not a request for change -- just a note to think about in the future

Reply:
Yep, definitely would've made my life easier :). Noted for next time.